package.json

```diff
@@ -12,38 +12,32 @@
   "files": [
     "README.md",
     "lib/*",
-    "lib/dynamodb/**",
-    "lib/services/**",
     "package.json"
   ],
   "scripts": {
-    "build": "yarn run build:babel && yarn run build:flow-copy-source && yarn run build:license-checker",
-    "build:babel": "babel src --out-dir lib --ignore test.js",
-    "build:flow-copy-source": "flow-copy-source --ignore test.js src lib",
+    "build": "yarn run build:license-checker && tsc",
+    "build:flowtypes": "find lib -type f -name '*.d.ts' -exec sh -c 'yarn flowgen --add-flow-header $1 -o ${1%.*.*}.js.flow' _ '{}' \\;",
     "build:license-checker": "license-checker --csv --out var/licenses.csv",
     "clean": "shx rm -rf lib",
+    "flow": "echo this is not a flow project",
     "git:commit": "git add package.json .yarn/versions && git commit -m v${npm_package_version}",
     "git:commit:publish": "git add package.json .yarn/versions/* && git commit -m v${npm_package_version}",
     "git:commit:version": "git add .yarn/versions/* && git commit -m 'Deferred version bump'",
     "git:push": "git push --follow-tags",
     "git:tag": "git tag -a v${npm_package_version} -m v${npm_package_version}",
     "npm:tag:remove": "yarn npm tag remove ${npm_package_name} $0",
-    "prepack": "yarn run build",
+    "prepack": "yarn run build && yarn run build:flowtypes",
```

Comment:
defer creation of flow types to the prepack to speed local builds

this is documented in the conversion steps